docs/introduction/introduction.md

```diff
@@ -1,68 +1,90 @@
-- Siembol prefers a declarative json language rather than a script language like Stellar. We consider declarative language with testing and validation less error prone and simpler to understand
-- Supporting oauth2/oidc  for authentication and authorisation in siembol UI
-- All siembol services can have multiple instances with authorisation based on oidc group membership. This allows multitenancy usage without the need to deploy multiple instances of siembol
-- We are planning to test and tune oauth2/oidc integration with popular identity providers
+### Integration with Other Systems – Siembol Response
 
-### Easy installation to try out with prepared docker images and helm charts
+- Easy way to integrate Siembol with other systems such as [Jira](https://www.atlassian.com/software/jira), [Cortex](https://www.cortex-ia.com/our-software/), [ELK](https://www.elastic.co/what-is/elk-stack), and [LDAP](https://ldap.com/).
+- Functionality to provide additional enrichments about an alert, such as [ELK](https://www.elastic.co/what-is/elk-stack) searches or [LDAP](https://ldap.com/) searches, with the option to filter the alert as part of an automatic incident response.
+- Plugin interface allowing for custom integration with other systems used in incident response.
+- We are planning to publish a collection of plugins that we are using internally at G-Research, while providing space for collecting plugins from the Siembol community.
+
+### Advanced Parsing Framework for Building Fault-Tolerant Parsers
+
+- Custom-built framework for normalizing logs (parsing) including chaining of extractors and transformations which allows the user to:
+   - extract JSON, CSV structures, key value pairs, timestamps.
+   - parse timestamps using standard formatters to an epoch form.
+   - transform messages by renaming fields, filtering fields, or even the option to filter the whole message.
+- Supporting use cases for advanced log ingestion using multiple parsers and a routing logic.
+- Supporting a generic text parser, syslog, BSD syslog, and NetFlow v9 binary parser.
+
+### Advanced Enrichment Component
+
+- Defining rules for selecting enrichment logic, joining enrichment tables, and defining how to enrich the processed log.
+
+### Configurations and Rules are Defined by a Web Application Siembol UI
+
+- All configurations are stored in JSON format and edited by web forms in order to avoid mistakes and speed up creation and learning time.
+- Configurations are stored in Git repositories.
+- Supporting high integrity use cases with protected GitHub main branches for deploying configurations.
+- Supporting validation and testing configurations. Moreover, Siembol UI supports creating and evaluating test cases.
+- Siembol prefers a declarative JSON language rather than a scripting language like [Stellar](https://stellar.tech/). We consider declarative language with testing and validation to be less error prone and simpler to understand.
```

Comment:
Metron stellar language link https://metron.apache.org/current-book/metron-stellar/stellar-common/index.html

Reply:
Changed link to Metron Stellar language link.